packages/plugin-core/src/commands/GotoNote.ts

```diff
@@ -33,7 +37,10 @@ type CommandOpts = {
 export { CommandOpts as GotoNoteCommandOpts };
 
 type CommandOutput =
-  | { note: NoteProps; pos?: Position; source?: string }
+  // When opening a note
+  | { type: "note"; note: NoteProps; pos?: Position; source?: string }
```

Comment:
since note types will soon be a thing, can we change the key here to `kind` to avoid confusion?

Reply:
also, can we use `nonNote` as the kind? prefer camelcase since we can use them in javascript without quoting